OpenKh.Tools.BbsMapStudio/ObjEntryController.cs

```diff
@@ -0,0 +1,101 @@
+using Microsoft.Xna.Framework.Graphics;
+using OpenKh.Common;
+using OpenKh.Engine.MonoGame;
+using OpenKh.Engine.Parsers;
+using OpenKh.Kh2;
```

Comment:
Is there a reason why you include kh2?

Reply:
Since this is basically a clone from the KH2 Map Studio, the tool still has some KH2 stuff it requires, but I should probably fix that before merging.

Reply:
I just checked and it isn't possible to remove this as the viewer heavily relies in a KH2 structure.

Reply:
Ok no problem.

Reply:
Consequently, how easy would it be to merge the two map studios and rename/begin the process of making it game agnostic? Not that I'm suggesting it be done necessarily, just curious.

Reply:
There're 2 ways we can go try to make map tools for individual games and a shared map tool project for common code or we can merge it together and have 1 tool and not like 5 map tools one for every game each.

Reply:
The problem with that is that you have to separate the game-centric features, like for example ARD spawns in KH2 and OLOs in BBS. Plus, you would need a way to detect which game has been selected, which is more hassle unless you add a "Load KH2 folder, Load BBS folder" as separate options.